cmd/create_github.go

```diff
@@ -64,6 +65,10 @@ func createGitHubAppE(command *cobra.Command, _ []string) error {
 		"GitHubEvent": fmt.Sprintf("%s://system.%s/github-event", scheme, rootDomain),
 	}
 
+	if _, err := os.Stat(path.Join("./pkg/github", "index.html")); err != nil {
+		return fmt.Errorf("Cannot find the template file. You need to run this command from the root directory of the repository.\n ")
```

Comment:
Just so that you know, errors in Go cannot start with an uppercase letter or end with a `.`

Reply:
Can I suggest this alternative? -> `cannot find template "index.html", run this command from the ofc-bootstrap repository`